app/assets/javascripts/components/NewsletterSticky.js

```diff
@@ -218,7 +228,26 @@ define([
     this.$el.removeClass(this.config.visibleClassName);
     this.closeButton.addClass(this.config.closedClassName);
     this._setListeners(false);
+    this.previousFocusElement.focus();
   };
 
+  /*
+  * Handle the tab event on the last and first focusable element -
+  * a tab loop is created until the user dismisses the dialog (accessibility recommendation).
+  * The keycode for the tab key is 9.
+   */
+  NewsletterSticky.prototype._handleLastElementTab = function(e) {
+    if (e.which === 9 && !e.shiftKey) {
```

Comment:
It might be worth setting a TAB_KEY var after $window on line 20 just so it's super clear in the future without having to look up the ASCII codes...

```
.
.
.
$newsletterStickForm,
newsletterCloseBoxCookieURL,
$window,
TAB_KEY = 9;
```